changelog.json

```
@@ -1,5 +1,18 @@
 {
   "releases": {
+    "3.4.9": [
+      "[Fixed] Fix drag and drop by addressing breaking Electron change - #19473",
```

```suggestion
      "[Fixed] Fix drag and drop by addressing breaking Electron change - #19473",
```
Fix was for beta. (Not applicable to prod since this is the electron ship to prod)